tests/test_onnx_conversion.py

```diff
@@ -30,25 +30,25 @@ class TestONNXConverter(unittest.TestCase):
 
     @classmethod
     def setUpClass(cls):
-
         X_bc, y_bc = load_breast_cancer(return_X_y=True)
         nrows = 15000
         X_bc: np.ndarray = X_bc[0:nrows]
         y_bc: np.ndarray = y_bc[0:nrows]
-        sklearn_model = RandomForestClassifier(n_estimators=10, max_depth=10)
-        sklearn_model.fit(X_bc, y_bc)
+        model = lgb.LGBMRegressor(n_estimators=3, min_child_samples=1)
```

Comment:
So RandomForestClassifier has an error, is there a similar issue in hummingbird?

Reply:
Here is a similiar issue with SGD classifier https://github.com/microsoft/hummingbird/issues/413, and I have opened a new issue for this problem https://github.com/microsoft/hummingbird/issues/474